src/rgw/jwt-cpp/jwt.h

```diff
@@ -233,6 +237,29 @@ namespace jwt {
 				} else
 					throw rsa_exception("at least one of public or private key need to be present");
 			}
+			rsa(const EVP_MD*(*md)(), const std::string& modulus, const std::string& exponent, const std::string& name) : md(md), modulus(modulus), exponent(exponent), alg_name(name)
+			{
+				string n_str = base64_decode_url(modulus);
+				string e_str = base64_decode_url(exponent);
+				unsigned char u_n[n_str.length()], u_e[e_str.length()];
+				std::copy(n_str.begin(), n_str.end(), u_n);
```

Comment:
it's not very important right now, but this copy could probably be avoided?

Reply:
BN_bin2bn gave a compilation error with char*